workloads/baseline.yml

```diff
@@ -11,6 +11,8 @@
     - vars/baseline.yml
   vars:
     workload_job: "baseline"
+  environment:
+    KUBECONFIG: "{{ kubeconfig_file }}"
```

Comment:
Should `kubeconfig_file` be set in ever vars/$WORKLOAD.yml file?

Reply:
@akrzos We set the kubeconfig_file in every workload vars file but if it's not set to ~/.kube/config the oc commands executed outside the scale-ci-workload pod won't work as it either looks at $HOME/.kube/config or KUBECONFIG env var or we need to use $oc --config <path-to-kubeconfig>. Setting the KUBECONFIG env var to the path mentioned in the workloads vars file i.e kubeconfig_file will be useful.

Reply:
Oops, I managed to miss that.  Curious though, is there a specific CI need for this at the moment or is this simply to help those who don't have kubeconfig in a standard place?  I can see this helping if you deploy multiple clusters from your local machine and want to run multiple workload repos for each cluster (Thus your kubeconfig file would be in different places for each cluster).  Is that the intended fix?

Reply:
That's one of the use case, we also need this for integrating Scale-CI into starter framework as the kubeconfig on starter clusters is not in the home directory.

Reply:
Got it. OK so we need to change the default as well since this will "override" where the kubeconfig "is".  Since tilde expansion will not occur inside the `environment` section for Ansible, we need to set the default to what is the most usualy case or most usualy un-automated case imo.  An example would be change it to `/root/.kube/config` since the `ORCHESTRATION_USER` default is `root` already and that is where the tilde would expand it to that case.  Thoughts?